thoth/storages/graph/query_result_base.py

```diff
@@ -43,3 +45,53 @@ def result(self):
     def count(self):
         """Query count result."""
         return self._count
+
+    def count_per_package(self) -> Dict[Tuple[str, str, str], int]:
```

Comment:
By adding these methods, the class becomes specific to Python packages - maybe rename to `PythonQueryResult` or something similar?

Reply:
Also, I'm not sure whether this approach is safe.

Reply:
I thought would give us clean code, also because more functions use similar output, so in this class we refactor the output of the queries to be in the way we need, but all queries give result using sqalchemy function, e.g. count() or all(). Alternatively I can take everything back :)

Reply:
Do you mean we have higher risk to break more methods at once?

Reply:
With this change we will need to make sure `PythonQueryResult` uses the structure of results so users of `PythonQueryResult` can call methods safely. Is this the case? - All methods which return `PythonQueryResult` object can safely call these methods?